Ports/SuperTuxKart/patches/0003-Fixup-Vulkan-SDL-platform-quirks.patch

```diff
@@ -0,0 +1,210 @@
+ #include <limits>
++#include <stdexcept>
+ 
+ #include "../source/Irrlicht/os.h"
+ #include "quaternion.h"
+diff --git a/lib/graphics_engine/src/ge_vulkan_fbo_texture.cpp b/lib/graphics_engine/src/ge_vulkan_fbo_texture.cpp
+index 9413a04bb00e0f19da1ab7ac104b8234b22f4bec..834a0b43b1e1a96bc1c7acb76910aa40d9aeee6c 100644
+--- a/lib/graphics_engine/src/ge_vulkan_fbo_texture.cpp
++++ b/lib/graphics_engine/src/ge_vulkan_fbo_texture.cpp
+@@ -6,6 +6,7 @@
+ 
+ #include <array>
+ #include <exception>
++#include <stdexcept>
+ 
+ namespace GE
+ {
+diff --git a/lib/graphics_engine/src/ge_vulkan_fbo_texture.hpp b/lib/graphics_engine/src/ge_vulkan_fbo_texture.hpp
+index ffc7fe9f831c9033c7951c3bdb3a788cec27513c..23d02a5b981e005a0ecbf66005844e7c478c97de 100644
+--- a/lib/graphics_engine/src/ge_vulkan_fbo_texture.hpp
++++ b/lib/graphics_engine/src/ge_vulkan_fbo_texture.hpp
+@@ -2,6 +2,7 @@
+ #define HEADER_GE_VULKAN_FBO_TEXTURE_HPP
+ 
+ #include "ge_vulkan_texture.hpp"
++#include <stdexcept>
+ 
+ namespace GE
+ {
+diff --git a/lib/graphics_engine/src/ge_vulkan_skybox_renderer.cpp b/lib/graphics_engine/src/ge_vulkan_skybox_renderer.cpp
+index 2543b16086ff134f2cf0eb435bb2986547aa25cb..b34340d7659b3af754183b6968d9cef5b7565a16 100644
+--- a/lib/graphics_engine/src/ge_vulkan_skybox_renderer.cpp
++++ b/lib/graphics_engine/src/ge_vulkan_skybox_renderer.cpp
+@@ -12,6 +12,7 @@
+ #include <array>
+ #include <cstdint>
+ #include <unordered_map>
++#include <stdexcept>
+ 
+ namespace GE
+ {
+diff --git a/lib/graphics_engine/src/ge_vulkan_texture.cpp b/lib/graphics_engine/src/ge_vulkan_texture.cpp
+index 500eb65fc6204bda5b21eaef25c1c9de1404005e..9f7f155092c0762b1f2e40f9800a04c1e0b9b628 100644
+--- a/lib/graphics_engine/src/ge_vulkan_texture.cpp
++++ b/lib/graphics_engine/src/ge_vulkan_texture.cpp
+@@ -9,6 +9,7 @@
+ #include "ge_vulkan_command_loader.hpp"
+ #include "ge_vulkan_features.hpp"
+ #include "ge_vulkan_driver.hpp"
++#include <stdexcept>
+ 
+ extern "C"
+ {
+diff --git a/lib/graphics_engine/src/ge_vulkan_texture_descriptor.cpp b/lib/graphics_engine/src/ge_vulkan_texture_descriptor.cpp
+index 4fd8a2e8800e6620fb5980097fefefdab8150ab8..46b671ffdbe55e9be6f35eeb7d05857aa12880bc 100644
+--- a/lib/graphics_engine/src/ge_vulkan_texture_descriptor.cpp
++++ b/lib/graphics_engine/src/ge_vulkan_texture_descriptor.cpp
+@@ -4,6 +4,7 @@
+ #include "ge_vulkan_driver.hpp"
+ #include "ge_vulkan_texture.hpp"
+ 
++#include <stdexcept>
+ #include <algorithm>
+ #include <exception>
+ 
+diff --git a/lib/irrlicht/source/Irrlicht/CIrrDeviceSDL.cpp b/lib/irrlicht/source/Irrlicht/CIrrDeviceSDL.cpp
+index 1dfef29db82f1f32bde84086d923dcea6e514329..b6aace563da5bd67ff52690eb8c43e192f13ee7f 100644
+--- a/lib/irrlicht/source/Irrlicht/CIrrDeviceSDL.cpp
++++ b/lib/irrlicht/source/Irrlicht/CIrrDeviceSDL.cpp
+@@ -128,10 +128,6 @@ CIrrDeviceSDL::CIrrDeviceSDL(const SIrrlichtCreationParameters& param)
+ 			SDL_VERSION(&Info.version);
+ 
+ 			// Switch doesn't support GetWindowWMInfo
+-#ifndef __SWITCH__

```

Comment:
Should `if !defined(__SWITCH__) && !defined(serenity)` here instead of deleting the whole block. same below with the if 0.